vtex-apps/minicart
# Feature/pass payment data to summary

#### What problem is this solving?

Passing paymentData as prop to Summary component to allow showing installments related information inside checkout-summary app. 

#### How to test it?

Go to [Workspace](https://paymentdata--tokstokio.myvtex.com/venda-corporativa#featured-products), add a product to cart and hover over minicart icon. Inspect SummaryContextProvider element with React DevTools to confirm paymentData being passed as prop.

#### Screenshots or example usage:

![image](https://user-images.githubusercontent.com/34974565/100485803-3d0cf900-30e0-11eb-99f7-21968780848a.png)
